connexion/operations/abstract.py

```diff
@@ -390,6 +385,17 @@ def function(self):
 
         return function
 
+    @property
+    def _request_response_decorator(self):
```

Comment:
Lifted from the removed `SecureOperation` as it's not actually related to security.